functions/swing_speed.h

```diff
@@ -26,4 +26,27 @@ using SwingSpeedX = SingleValueAdapter<SwingSpeedSVF<MAX>>;
 template<int MAX>
 using SwingSpeed = SwingSpeedX<Int<MAX>>;
 
+// Usage: SwingAcceleration<MAX>
+// Returns 0-32768 based on swing acceleration
+// MAX defaults to 150
+// returned value: INTEGER
+template<class MAX = Int<130>>
+class SwingAccelerationSVF {
+ public:
+  void run(BladeBase* blade) { max_.run(blade); }
+ int calculate(BladeBase* blade) {
+   max_.run(blade);
```

Comment:
No need to call max_.run() again. (It's already done in SwingAccelerationSVF::run() above)

Reply:
remove duplication in SwingSpeed as well